src/main/java/hw10/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,28 @@
+package hw10;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class FileLoggerConfigurationLoader {
+    public FileLoggerConfiguration load(String filePath) throws IOException {
+        BufferedReader reader = new BufferedReader(new FileReader(filePath));
+        String line;
+        FileLoggerConfiguration configuration = new FileLoggerConfiguration();
+        while ((line = reader.readLine()) != null) {
+            String[] parts = line.split("-");
+            String key = parts[0].trim();
+            String value = parts[1].trim();
+            switch (key) {
+                case "FILE" -> configuration.setFile(new File(value));
+                case "LEVEL" -> configuration.setLoggingLevel(LoggingLevel.valueOf(value));
+                case "SIZE" -> configuration.setMaxFileSize(Long.parseLong(value));
+                case "FORMAT" -> configuration.setMessageFormat(value);
+                default -> System.err.println("Unknown configuration key: " + key);
+            }
+        }
+        reader.close();
```

Comment:
Using try-with-resource allows you not to close the resource manually as it is done for you automatically.